PartymodeAutostart.py

```diff
@@ -35,8 +32,11 @@ def __init__(self):
         if self.startupPartyMode and self._viewCountdown():
             self.runPartyMode()
 
-        while not xbmc.abortRequested:
-            xbmc.sleep(1000)
+        KODIMONITOR = xbmc.Monitor()
+        KODIMONITOR.waitForAbort( 5 )            
+
+        # while not xbmc.abortRequested:
+        #     xbmc.sleep(1000)
```

Comment:
I'm not sure do the same thing

Reply:
the kodi api changed. fix taken from here https://forum.kodi.tv/showthread.php?tid=354577